src/lib/dnssd/minimal_mdns/responders/IP.cpp

```diff
@@ -21,12 +21,20 @@
 namespace mdns {
 namespace Minimal {
 
+using namespace chip::Inet;
+
 void IPv4Responder::AddAllResponses(const chip::Inet::IPPacketInfo * source, ResponderDelegate * delegate,
                                     const ResponseConfiguration & configuration)
 {
-    chip::Inet::IPAddress addr;
-    for (chip::Inet::InterfaceAddressIterator it; it.HasCurrent(); it.Next())
+    IPAddress addr;
+    for (InterfaceAddressIterator it; it.HasCurrent(); it.Next())
     {
+        if (it.GetFlags().HasAny(InterfaceAddressIterator::Flags::kNotFinal, InterfaceAddressIterator::Flags::kTemporary,
```

Comment:
Add comment here and in IPv6 that we will not advertise addresses that are not current and stable